corehq/form_processor/utils/general.py

```diff
@@ -29,6 +29,8 @@ def should_use_sql_backend(domain_object_or_name):
 
 
 def should_use_sql_backend_in_prod(domain_object_or_name):
+    # the only reason this method is "public" is so it can be called directly by a handful of tests.
```

Comment:
in the past i've still kept functions prefixed with `_` even if they were just used in tests. i'm not sure if that's the correct thing to do and might say something about the level of testing